src/Common/FitData.m

```diff
@@ -105,7 +106,7 @@
 
         else
             if getappdata(h,'canceling');  break;  end  % Allows user to cancel
-            waitbar(ii/numVox, h, sprintf('Fitting voxel %d/%d', ii, numVox));
+            waitbar(ii/numVox, h, sprintf('Fitting voxel %d/%d (%d errors)', ii, numVox, fitFailedCounter));
```

Comment:
I tried with several examples (`ir_t1` and `mwf`), but could not hit exception at line 121 yet :) After reading your changes, don't think that it is going to be a problem, but would be useful if you cant hint me which model to use to hit the exception. 

Regarding wording, is there any term we can use here instead of `errors` ?

Reply:
@agahkarakuzu in order to get errors, I temporarily changed a the model fitting method to introduce a bug.
Concerning the wording, I don't have a preference. exception/singularity/failed...